### PR TITLE
Add json-pretty-print-enabled cli option

### PR DESCRIPTION
**Pull request by gtebrean**
Update the CLI option docs according to last changes done in this PR https://github.com/hyperledger/besu/pull/5766

### CODE REVIEW

**Comment by alexandratran** [comment]
@gtebrean Can you make this change to the content in the `docs/` folder? That ensures the change is included in the latest version of the docs and future released doc versions.

**Comment by gtebrean** [comment]
> @gtebrean Can you make this change to the content in the `docs/` folder? That ensures the change is included in the latest version of the docs and future released doc versions.

ok, I see now, please check

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -3532,6 +3532,38 @@ rpc-ws-port="6174"
...
+# Example
+
+```bash
+--json-pretty-print-enabled=true
+```
+
+# Environment variable
+
+```bash
+JSON_PRETTY_PRINT_ENABLED=true
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:3554]:
```suggestion
BESU_JSON_PRETTY_PRINT_ENABLED=true
```

Should this be prefixed with `BESU_` like the other environment variables?

File: docs/public-networks/reference/cli/options.md [hunk 2]
```diff
@@ -3532,6 +3532,38 @@ rpc-ws-port="6174"
...
+
+# Configuration file
+
+```bash
+json-pretty-print-enabled=true
+```
+
+<!--/tabs-->
+
+Enables or disables the pretty-print the output for HTTP and Websockets responses. The default is `false`.
```
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:3565]:
```suggestion
Enables or disables the pretty-print output for HTTP and WebSocket responses. The default is `false`.
```

**Review by alexandratran — COMMENTED** [review]
Minor comments.

**Review by alexandratran — APPROVED** [review]

**Pull request merged**